proposals/Configs.sol

```diff
@@ -59,6 +73,66 @@ abstract contract Configs is Test {
         temporalGovDelay[OPTIMISM_SEPOLIA_CHAIN_ID] = 0 days;
         temporalGovDelay[LOCAL_CHAIN_ID] = 0 days;
         temporalGovDelay[BASE_SEPOLIA_CHAIN_ID] = 0 days;
+
+        /// Initialize oracle configurations for Base
```

Comment:
keep these inside the x35 proposal script

Reply:
missing some - find them in moonwell app and use basescan on `ChainlinkOrace.getFeed(symbol)`